crates/libcgroups/src/v2/devices/controller.rs

```diff
@@ -57,8 +63,8 @@ impl Devices {
 
         // Increase `ulimit -l` limit to avoid BPF_PROG_LOAD error (#2167).
         // This limit is not inherited into the container.
-        bpf::bump_memlock_rlimit()?;
```

Comment:
I actually liked the `bpf::` namespacing around here and think I could reintroduce it by adding `as bpf` above.

Reply:
I have reintroduced this